packages/react/src/FronteggProvider.tsx

```diff
@@ -5,18 +5,39 @@ import { BrowserRouter } from 'react-router-dom';
 import { useHistory } from 'react-router';
 import { ContextHolder, RedirectOptions } from '@frontegg/rest-api';
 
-export type FronteggProviderProps = FronteggConfigOptions;
-type ConnectorProps = FronteggProviderProps;
+export type FronteggProviderProps = FronteggConfigOptions & {
+  overrideHistory?: {
```

Comment:
Don't call it overrideHistory. Call it `history`
In the docs we will explain that you can pass your own history and override

Reply:
Done